### DISCUSSION TITLE
I've waited for 2 week for processing my application. Is this normal ?

### DISCUSSION THREAD

**Discussion by dnthang** [community/community#70168] [GitHub Education] [answered]
### Select Topic Area

Product Feedback

### Body

![image](https://github.com/community/community/assets/138576063/9f41de67-61b5-4cba-9b95-40298ddd8019)

**Comment by andrej-kocijan** [comment]
In my experience the amount of time can vary quite a bit, don't think 2 weeks is a reason to worry for now. If you do find yourself stuck on pending you can get in contact with the staff via https://support.github.com/request

**Reply by dnthang** [reply]
I check status via the link github support but none status was shown instead it's navigate me the form at the beginning. What should i do next. Thanks
![image](https://github.com/community/community/assets/138576063/cdb940f6-fe2f-447b-8c22-4ccc5ef4b6d0)

**Reply by andrej-kocijan** [reply, marked as answer]
If you do not see your applications [here](https://education.github.com/discount_requests/application) you should consider re-applying or [contacting help](https://support.github.com/contact/education)

**Reply by dnthang** [reply]
Thank you so much, I got approved !!!!

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @dnthang , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Reply by dnthang** [reply]
thank you !!
